spec/acceptance/authentication_spec.rb

```diff
@@ -1,9 +1,11 @@
+# frozen_string_literal: true
+
 require 'spec_helper_acceptance'
 require 'uri'
 
 context 'authenticated download' do
   let(:source) do
-    URI.escape("http://httpbin.org/basic-auth/user/#{password}")
+    CGI.escape("http://httpbin.org/basic-auth/user/#{password}")
```

Comment:
`CGI.escape` doesn't play well with `URI.regexp` (https://github.com/voxpupuli/puppet-archive/blob/master/lib/puppet/type/archive.rb#L127-L129)

I'm wondering if switching to normal regexp is an option?

Reply:
I am fine with whatever works :D